packages/terra-list/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed the console error appearing in the terra-menu examples while keyboard navigation.
```

Comment:
What are the errors?

Reply:
Following is the error getting while navigating through arrow key:- 
 Uncaught TypeError: Cannot read properties of undefined (reading 'focus')

Reply:
We can rephrase this to:

`Fixed _property is undefined_ error while navigating with a keyboard.`